db/seeds/playbooks/professional.seeds.rb

```diff
@@ -214,78 +214,35 @@
     Improvement.find_or_create_by!(improvement)
   end
 
+  #####
   # Gear
-  [
-    {
-      name: 'flak vest',
-      armor: 1,
-      tag_list: %w[hidden]
-    },
-    {
-      name: 'combar armor',
-      armor: 2,
-      tag_list: %w[heavy]
-    },
-    {
-      name: 'Assault rifle',
-      harm: 3,
-      tag_list: %w[far area loud reload]
-    },
-    {
-      name: 'Grenade launcher',
-      harm: 4,
-      tag_list: %w[far area messy loud reload]
-    },
-    {
-      name: 'Sniper Rifle',
-      harm: 4,
-      tag_list: %w[far]
-    },
-    {
-      name: 'Sniper Rifle',
-      harm: 4,
-      tag_list: %w[far]
-    },
-    {
-      name: 'Grenades',
-      harm: 4,
-      tag_list: %w[close area messy loud]
-    },
-    {
-      name: 'Submachine gun',
-      harm: 3,
-      tag_list: %w[close area loud reload]
-    },
-    {
-      name: '.38 revolver',
-      harm: 2,
-      tag_list: %w[close reload loud]
-    },
-    {
-      name: '9mm',
-      harm: 2,
-      tag_list: %w[close loud]
-    },
-    {
-      name: 'Hunting rifle',
-      harm: 2,
-      tag_list: %w[far loud]
-    },
-    {
-      name: 'Shotgun',
-      harm: 2,
-      tag_list: %w[close messy]
-    },
-    {
-      name: 'Big knife',
-      harm: 1,
-      tag_list: %w[hand]
-    }
-  ].each do |gear_attrs|
+  #####
+
+  # Array of ID's for The Professional
+  # Armor: You get either a
+  #   flak vest (1-armour hidden) - 31
+  # or
+  #   combat armour (2-armour heavy) - 21
+  # for protection.
+  #   Serious weapons (pick one):
+  # Assault rifle (3-harm far area loud reload) - 4
+  # Grenade launcher (4-harm far area messy loud reload) - 36
+  # Sniper rifle (4-harm far) - 69
+  # Grenades (4-harm close area messy loud) - 37
+  # Submachine gun (3-harm close area loud reload) - 74
+
+  #   Normal weapons (pick two):
+  # .38 revolver (2-harm close reload loud) - 2
+  # 9mm (2-harm close loud) - 3
+  # Hunting rifle (2-harm far loud) - 46
+  # Shotgun (3-harm close messy) - 62
+  # Big knife (1-harm hand) - 12
+
+  # serious or normal can be searched by the description column for each gear
+  [31, 21, 4, 36, 69, 37, 74, 2, 3, 46, 62, 12].each do |gear_id|
```

Comment:
This list of ids will be different in each environment, especially production, which already has gear loaded. Name and Harm would be a better way to associate this.

Reply:
We discussed this elsewhere, but just to have a record of what was decided in case we need to look back over it:
Unfortunately, searching by harm and name won't work, because we have entries that share name and harm. The solution we came up with was to populate the through table through a reference in gear to what playbooks they belong in. Initial plan is create a new column referencing the playbook names in the gear table, then rewrite the seed files to populate in that manner.

Reply:
Addressed in commits d886634 and f886cf9